pkg/graveler/graveler.go

```diff
@@ -249,6 +252,10 @@ type Repository struct {
 	InstanceUID string
 }
 
+type RepositoryMetadata struct {
+	LastImportTimestamp time.Time
```

Comment:
I would _really_ like to have a repository metadata mechanism that handles a (small-ish!) key-value map.  There are lots of metadata items that I could add -- and not all of them make sense within lakeFS, some only for users.  For instance: I would expect to control with a repository metadata values:
* where _hooks_ are stored;
* "stuff" about OTFs;
* an external URL for linking commits from the Airflow provider...

I think of it as similar to tags on AWS objects: you get the tags, and can use them as you see fit.